examples/kubernetes/storageclass-secret.yaml

```diff
@@ -8,7 +8,7 @@ mountOptions:
     - nfsvers=4.1
     - sec=sys
 parameters:
-  profile: "tier-5iops"                 # The VPC Storage profile used.
+  profile: "dp2"                 # The VPC Storage profile used.
   # iopsPerGB: "5"                      # The IOPS per Gigabyte that is supported for this profile. This is just for the user info.
   sizeRange: "[10-2000]GiB"             # The size range in GiB that is supported. The user will specify a size on the PVC
```

Comment:
Is this range applicable for dp2 Profile ?

Reply:
Done